force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -357,13 +467,20 @@ public with sharing class ClassesObjectsMethods {
     public static List<Person> createMultiplePersons(List<String> names, Integer age) {
 
         // Loop through each name in the provided list of names
-            // Create a new Person object for each name
+        // Create a new Person object for each name
+
+        // Add the new Person object to the list of Person objects
+
+            List<Person> persons = new List<Person>();
+
+            for (String listNames : names){
+                Person newPerson = new Person(listNames, age);
+                persons.add(newPerson);
+            }
```

Comment:
Good work. I would just add the person to the list directly in the loop and skip the variable.